.ci/wercker.sh

```diff
@@ -447,6 +447,17 @@ no-warning-imports-java-design-patterns)
   fi
   ;;
 
+validate-ci-temp-empty)
+  fail=0
+  if [ "$(ls -A .ci-temp)" ]; then
+    find ".ci-temp" -mindepth 1 -type d -exec echo "Found directory in .ci-temp: " {} \;
+    find ".ci-temp" -mindepth 1 -type f -exec echo "Found file in .ci-temp: " {} \;
```

Comment:
instead of these `find` ... why not simply `ls -A .ci-temp` ?
is you do validation by this commands it is always good to print output of it to let us see it.

Reply:
Very good point.  Thank you.  Fixed.